.github/workflows/alpha-team-two-workflow.yml

```diff
@@ -25,6 +25,6 @@ jobs:
         run: |          
           docker build -t alpha-team-two .
 
-      - name: Run Docker image
-        working-directory: ./group-work/AlphaTeamTwo/
-        run: docker run alpha-team-two
+   #   - name: Run Docker image
+   #     working-directory: ./group-work/AlphaTeamTwo/
+        #run: docker run alpha-team-two
```

Comment:
Please do not merge in commented code. I believe the best route here would be to remove lines 28-30

Reply:
aside from this, the PR looks good to go

Reply:
removed commented code